client/src/components/header.jsx

```diff
@@ -0,0 +1,84 @@
+import styled, { css } from 'styled-components';
+import React, {Component} from 'react'
+import DollarSignsAndCategories from './dollarsignsandCategories.jsx';
+import Categories from './categories.jsx';
+import StarsAndReviews from './stars.jsx'
+
+const Title = styled.h1`
+  font-size: 30px;
+  color: #333;
+  margin: 0 6px;
+`;
+
+const BizWrapper = styled.div`
+  width: 960px;
+  margin: 0 auto;
+  padding: 0 15px;
+  font-family: Helvetica Neue,Helvetica,Arial,sans-serif; 
+  font-weight: 700:
+`
+const ClaimedText = styled.div`
+  font-size: 14px;
+  display: inline-block;  
+`
+
+
+
+
+class Header extends React.Component {
+  constructor(props){
+    super (props)
+
+    this.state = {
+      businessData: {}
+    }
+  }
+
+  componentDidMount(){
+    // function that gets all search params
+    let parseQueryString = () => {
+      let str = window.location.search;
+      let objURL = {};
+      str.replace(
+          new RegExp( "([^?=&]+)(=([^&]*))?", "g" ),
+          ( $0, $1, $2, $3 ) => {
+              objURL[ $1 ] = $3;
+          }
+      );
+      return objURL;
+    };
+    let params = parseQueryString();
+    let idParam = params.id;
+    $.ajax({
+      url: `http://127.0.0.1:3004/business/${idParam}`,
+      success: (data) => {
+        let businessData = JSON.parse(data);
+        console.log("Data for this business", businessData);
+        this.setState({businessData: businessData})
+      }
+    })
+  }
+
+  render() {
+    console.log("inital state", this.state);
+    
+    return (
+      <BizWrapper>
+        <Title>
+          {this.state.businessData.name} 
+              {this.state.businessData.claimed === true &&
+                <ClaimedText> 
+                <i className="fas fa-check-circle"></i> 
```

Comment:
classNames seem vague, no sure what they are referring to or their purpose

Reply:
className is the JSX way of declaring a class, since 'class' is a keyword in javascript. 

line 71 here uses font awesome (check out https://fontawesome.com/) to get certain symbols and stuff.